test/index.js

```diff
@@ -1193,14 +1237,15 @@ describe('def', function() {
 
     throws(function() { $.RecordType({x: /XXX/, y: /XXX/, z: $.Any}); },
            TypeError,
-           'Invalid values\n' +
+           'Invalid value\n' +
            '\n' +
-           'The argument to ‘RecordType’ must be an object mapping field name to type.\n' +
+           'RecordType :: StrMap Type -> Type\n' +
+           '                     ^^^^\n' +
+           '                      1\n' +
            '\n' +
-           'The following mappings are invalid:\n' +
+           '1)  /XXX/ :: RegExp\n' +
```

Comment:
ah i see, agree on this. Maybe in future we could update the `def` to have same results (listing all issues instead of just first one).